bodhi/server/models.py

```diff
@@ -3391,9 +3417,7 @@ def has_stable_comment(self):
         """
         for comment in self.comments_since_karma_reset:
             if comment.user.name == 'bodhi' and \
-               comment.text.startswith('This update has reached') and \
-               'and can be pushed to stable now if the ' \
-               'maintainer wishes' in comment.text:
+               comment.text == 'This update can be pushed to stable now if the maintainer wishes':
```

Comment:
>     1. We can use value from `testing_approval_msg` config setting instead of hardcoding str. This way we can fix #1014.
> 

Yes good idea, I ll change that.

>     2. It looks a little bit backward incompatible, because current comment text is different. On the other hand, it turns out that this change will only cause Bodhi to comment about update readiness twice.

Yes there is a risk that for a some updates Bodhi will comment twice, to me this is acceptable. What do @bowlofeggs and @abompard think about that ?

Reply:
What about changing the first test to `.startswith('This update ')` and the second to `'can be pushed to stable now if the maintainer wishes'`? This way it'll find the new version of the comment too.

Reply:
@abompard that works for me :-)